.github/workflows/go.yml

```diff
@@ -0,0 +1,43 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
May as well convert this repo to main before you merge this and then just squash that here, too?

Reply:
Done did: e3a8805